src/cobbler_tftp/settings/schemas/v1_0.py

```diff
@@ -0,0 +1,68 @@
+"""
+Configuration for version 1.0
+"""
+
+from pathlib import Path
+from typing import Dict, Union
+
+from schema import Optional, Schema, SchemaError
+
+settings_schema: Schema = Schema(
+    {
+        Optional("schema"): float,
+        Optional("auto_migrate_settings"): bool,
+        Optional("is_daemon"): bool,
+        Optional("cobbler"): {
+            Optional("uri"): str,
+            Optional("username"): str,
+            Optional("password"): str,
+            Optional("password_file"): str | Path,
+        },
+    }
+)
+
+
+def validate_config(
+    settings_dict: Dict[str, Union[str, Path, Dict[str, Union[str, Path]]]]
+) -> bool:
+    """
+    Validates the given dictionary of configuration parameters to the reference ``schema``.
+
+    :param settings_dict: The dictionary of configuration parameters to validate
+    :return bool: True/False depending on whether the dicts match or not
+    """
+    if settings_dict == {} or settings_dict == None:
+        return False
+
+    try:
+        settings_schema.validate(settings_dict)
+    except SchemaError as exc:
+        print(f"{exc}: No valid configuration detected! (validate_config_error)")
+        return False
+    return True
+
+
+def normalize_config(
```

Comment:
```suggestion
def normalize(
```

To stay in line with Cobbler I would kindly ask that you remove the `config` suffix from this method.

Reply:
resolved. See `validate` discussion